packages/cli-e2e/docker/config/config.yaml

```diff
@@ -0,0 +1,84 @@
+#
+# This is the config file used for the docker images.
```

Comment:
I would cleanup the generic verdacio comments/commented out code. 

I assume you copied from their documentation ? Maybe a link would be better. 

And then, I would keep just the things that we actually use in this config, with the important/specific stuff that we need commented.